src/transformers/models/clvp/tokenization_clvp.py

```diff
@@ -0,0 +1,546 @@
+                list of strings (pretokenized string). If the sequences are provided as list of strings (pretokenized),
+                you must set `is_split_into_words=True` (to lift the ambiguity with a batch of sequences).
+            return_input_ids_with_special_tokens (`bool`, *optional*, defaults to False):
+                Whether to return `input_ids` with special tokens alongside `input_ids`. This is used by the
+                [`ClvpConditioningEncoder`] instead of `input_ids`. Corresponding `attention_mask` will be provided if
+                needed.
+        """
+        encodings = super().__call__(
+            text=text,
+            text_pair=text_pair,
+            text_target=text_target,
+            text_pair_target=text_pair_target,
+            add_special_tokens=add_special_tokens,
+            padding=padding,
+            truncation=truncation,
+            max_length=max_length,
+            stride=stride,
+            is_split_into_words=is_split_into_words,
+            pad_to_multiple_of=pad_to_multiple_of,
+            return_tensors=return_tensors,
+            return_token_type_ids=return_token_type_ids,
+            return_attention_mask=return_attention_mask,
+            return_overflowing_tokens=return_overflowing_tokens,
+            return_special_tokens_mask=return_special_tokens_mask,
+            return_offsets_mapping=return_offsets_mapping,
+            return_length=return_length,
+            verbose=verbose,
+            **kwargs,
+        )
+
+        # used by only [`ClvpConditioningEncoder`].
+        if return_input_ids_with_special_tokens:
+            encodings_with_special_tokens = super().__call__(
+                text=text,
+                text_pair=text_pair,
+                text_target=text_target,
+                text_pair_target=text_pair_target,
+                add_special_tokens=True,
+                padding=padding,
+                truncation=truncation,
+                max_length=max_length,
+                stride=stride,
+                is_split_into_words=is_split_into_words,
+                pad_to_multiple_of=pad_to_multiple_of,
+                return_tensors=return_tensors,
+                return_token_type_ids=return_token_type_ids,
+                return_attention_mask=return_attention_mask,
+                return_overflowing_tokens=return_overflowing_tokens,
+                return_special_tokens_mask=return_special_tokens_mask,
+                return_offsets_mapping=return_offsets_mapping,
+                return_length=return_length,
+                verbose=verbose,
+                **kwargs,
+            )
+
+            encodings["input_ids_with_special_tokens"] = encodings_with_special_tokens["input_ids"]
+            encodings["attention_mask_with_special_tokens"] = encodings_with_special_tokens["attention_mask"]
+
+        return encodings
+
+    def _decode(
```

Comment:
not 100% sure I see why this is overloaded ? both spaces_between_special_tokens and ``clean_up_tokenization_spaces` can be saved

Reply:
We don't need to overload it, sorry . 
Removed it in the recent commit.